std/node/_fs/_fs_mkdtemp_test.ts

```diff
@@ -0,0 +1,133 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
```

Comment:
If this uses code from Node please add the copyright too? (I think there is already process for that?)